demos.js

```diff
@@ -266,7 +266,7 @@ const allDemos = [
         "group": "ate bit",
         "youtube": "jhXHZ9ZfsKI",
         "date": "2009-08-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00053/00053646.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/da/77/7948.322231.jpg",
```

Comment:
I think the available screenshots don't do justice to this prod. We might want to take make one ourselves at some point.